apps/codeforafrica/package.json

```diff
@@ -42,8 +42,9 @@
     "@mui/material": "^5.8.5",
     "@mui/utils": "^5.8.4",
     "fast-equals": "^4.0.1",
-    "frontmatter-markdown-loader": "^3.7.0",
+    "gray-matter": "^4.0.3",
     "js-yaml": "^4.1.0",
+    "marked": "^4.0.18",
```

Comment:
Why not use `remark`? [Netlify CMS](https://www.netlifycms.org/docs/beta-features/#remark-plugins) and Next.js examples seem to prefer it.

Reply:
For this, I think it's just a preference. But `marked` seems to be more actively maintained and it's standalone not requiring other plugins e.g `remark-html`